### PR TITLE
fixed issue #396

### PR DESCRIPTION
**Pull request by sajarindider**
@B3zo0 sorry for the annoyance, this is my new attempt at solving this. I'm merging it to the develop branch of the remote as you requested. Let me know if you want to me do any changes.

### CODE REVIEW

File: packages/sn-client-utils/package.json [hunk 1]
```diff
@@ -33,5 +33,7 @@
   "bugs": {
     "url": "https://github.com/SenseNet/sn-client/issues"
   },
-  "homepage": "https://sensenet.com"
-}
+  "homepage": "https://sensenet.com",
+  "dependencies": {
+    "tslib": "^1.9.3"
+  }
```
Comment (B3zo0) [inline packages/sn-client-utils/package.json]:
```suggestion
  }
}
```
A closing curly brace is missing from here

File: packages/sn-default-content-types/package.json [hunk 2]
```diff
@@ -31,5 +31,7 @@
   },
   "author": "Sense/Net <snteam@sensenet.com> (http://sensenet.com/)",
   "license": "GPL-2.0",
-  "homepage": "https://sensenet.com"
-}
+  "homepage": "https://sensenet.com",
+  "dependencies": {
+    "tslib": "^1.9.3"
+  }
```
Comment (B3zo0) [inline packages/sn-default-content-types/package.json]:
```suggestion
  }
}
```

File: packages/sn-query/package.json [hunk 3]
```diff
@@ -31,5 +31,7 @@
   "bugs": {
     "url": "https://github.com/SenseNet/sn-client/issues"
   },
-  "homepage": "https://sensenet.com"
-}
+  "homepage": "https://sensenet.com",
+  "dependencies": {
+    "tslib": "^1.9.3"
+  }
```
Comment (B3zo0) [inline packages/sn-query/package.json]:
```suggestion
  }
}
```

**Comment by B3zo0** [comment]
Hi @sajarindider, thanks. Few minor mistakes I've suggested change for other than that looks good. Lets see a green ci and I will merge it in. Also did you fill out our [form](https://www.sensenet.com/Hacktoberfest2019)?

**Comment by sajarindider** [comment]
@B3zo0 seems like there was an additional typo with one of the dependencies I added. Just fixed it. Is it good enough to merge now?

**Pull request merged**